stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -949,30 +953,74 @@ void create(Map<String, Object> params,
         }
 
         @Override
-        protected ResponseWrapper doInBackground(Void... params) {
+        protected ResponseWrapper<Source> doInBackground(Void... params) {
             try {
                 final Source source = mApiHandler.createSource(
                         mSourceParams,
                         mPublishableKey,
                         mStripeAccount
                 );
-                return new ResponseWrapper(source);
+                return new ResponseWrapper<>(source);
             } catch (StripeException stripeException) {
-                return new ResponseWrapper(stripeException);
+                return new ResponseWrapper<>(stripeException);
             }
         }
 
         @Override
-        protected void onPostExecute(@NonNull ResponseWrapper responseWrapper) {
-            if (responseWrapper.source != null) {
-                mSourceCallback.onSuccess(responseWrapper.source);
+        protected void onPostExecute(@NonNull ResponseWrapper<Source> responseWrapper) {
+            if (responseWrapper.result != null) {
+                mSourceCallback.onSuccess(responseWrapper.result);
```

Comment:
maybe out of scope, but I think we should be doing the same thing we're doing for tokens:

```
            } else {
                mCallback.onError(new RuntimeException(
                        "Somehow got neither a token response or an error response"));
            }
```